src/controllers/features/unincorporated/addressCorrespondanceSelectorController.ts

```diff
@@ -94,6 +94,9 @@ export const post = async (req: Request, res: Response, next: NextFunction) => {
                     acspData.applicantDetails = applicantDetails;
                     //  save data to mongodb
                     await acspDataService.saveAcspData(session, acspData);
+                } else {
```

Comment:
Here we are just comparing postcode. if (applicantDetails.correspondenceAddress?.postalCode === acspData.registeredOfficeAddress?.postalCode) .
I think, we need to compare 1st line of address as well?

Reply:
Makes sense. But I think we should then compare the whole address actually. Let me see if there is any easy way of doing this

Reply:
Added the check to compare full address.